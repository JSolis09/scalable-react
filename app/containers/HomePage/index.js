import React from 'react';

/* eslint-disable react/prefer-stateless-function */
export default class HomePage extends React.PureComponent {
  render() {
    return (
      <div>
        Home Page!!
      </div>
    );
  }
}
