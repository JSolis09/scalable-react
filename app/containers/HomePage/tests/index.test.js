import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../index';

describe('<HomePage />', () => {
  it('should render correctly', () => {
    const homeComponent = shallow(<HomePage />);
    expect(homeComponent).toMatchSnapshot();
  });
});
