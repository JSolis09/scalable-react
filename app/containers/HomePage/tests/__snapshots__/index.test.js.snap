// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render correctly 1`] = `
<div>
  <withReducer(withSaga(Connect(NavigationContainer))) />
  <withReducer(withSaga(Connect(LinkListContainer))) />
</div>
`;
